include/crypto/err.h

```diff
@@ -15,4 +15,7 @@ void err_cleanup(void);
 int err_shelve_state(void **);
 void err_unshelve_state(void *);
 
+char *openssl_buf2hexstr_sep(const unsigned char *buf, long buflen, char sep);
+unsigned char *openssl_hexstr2buf_sep(const char *str, long *buflen,
+                                      const char sep);
```

Comment:
This is wrongly placed, as these are not ERR functions per se.  Please move these declarations to `cryptlib.h` (side note: that header should never have been moved to `include/internal`, as it was meant as declarations of libcrypto private functions)